tests/func/test_get.py

```diff
@@ -187,17 +152,17 @@ def test_non_cached_output(tmp_dir, erepo_dir):
 
 # https://github.com/iterative/dvc/pull/2837#discussion_r352123053
 def test_absolute_file_outside_repo(tmp_dir, erepo_dir):
-    with pytest.raises(PathMissingError):
+    with pytest.raises(FileNotFoundError):
```

Comment:
Part of generalizing tree exceptions. We need to make FileNotFoundError just have a more useful message with repo addressing (e.g. gh url if it was cloned).